MBBSEmu.Tests/ExportedModules/Majorbbs/intdos_Tests.cs

```diff
@@ -0,0 +1,319 @@
+﻿using MBBSEmu.CPU;
+using MBBSEmu.Memory;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace MBBSEmu.Tests.ExportedModules.Majorbbs
+{
+    public class intdos_Tests : ExportedModuleTestBase
+    {
+        private const int INTDOS_ORDINAL = 355;
+
+        [Fact]
+        public void intdos_0x19_Test()
+        {
+            //Reset State
+            Reset();
+
+            //Set Argument Values to be Passed In
+            var testRegisters = new CpuRegisters {AH = 0x19};
+
+            //Allocate some memory to hold the test data
+            var testRegistersArrayData = testRegisters.ToRegs();
+            var testRegistersPointer = mbbsEmuMemoryCore.AllocateVariable(null, (ushort) testRegistersArrayData.Length);
+            mbbsEmuMemoryCore.SetArray(testRegistersPointer, testRegistersArrayData);
+
+            //Execute Test
+            ExecuteApiTest(HostProcess.ExportedModules.Majorbbs.Segment, INTDOS_ORDINAL,
+                new List<IntPtr16> {testRegistersPointer, testRegistersPointer});
+
+            //Data returned in memory, so we need to reload testRegisters from memory
+            testRegisters.FromRegs(mbbsEmuMemoryCore.GetArray(testRegistersPointer,
+                (ushort) testRegistersArrayData.Length));
+
+            //Verify Results
+            Assert.Equal(2, testRegisters.AL);
+        }
+
+        [Fact]
+        public void intdos_0x1A_Test()
+        {
+            //TODO
+        }
+
+        [Fact]
+        public void intdos_0x25_Test()
```

Comment:
could you add the functional name in the test as well, for readability?

intdos_0x25_SetInterruptVector_Test()?